test/models/medical_test.rb

```diff
@@ -9,6 +9,24 @@ class MedicalTest < ActiveSupport::TestCase
     refute Medical.new(date: Date.today - 2.years).good?
   end
 
+  test "medical from valid from trip start to end date is good for trip" do
```

Comment:
I'm not sure I understand the test name. Maybe `a medical that is valid from a trip's start to end date is good for the trip`